config/initializers/rack-attack.rb

```diff
@@ -2,33 +2,36 @@
   req.ip if req.post? && req.path.start_with?("/users/sign_in")
 end
 
-Rack::Attack.throttle("searches/ip", limit: 15, period: 15.minutes) do |req|
-  # don't throttle requests with a q, because it's more likely to be a real user
-  next if req.params['q'].present?
+# throttle bot-like search queries if configured
+if ActiveModel::Type::Boolean.new.cast(ENV.fetch("THROTTLE_SEARCHES", Settings.throttle_searches))
```

Comment:
How about using `SETTINGS__THROTTLE_SEARCHES` as the env var. Then we can just do `Settings.throttle_searches`. The config gem can cast to boolean and load from ENV vars if properly configured.

Reply:
Testing using the console and this worked exactly how I'd expect...pretty cool. made the change